test/urlencoded.js

```diff
@@ -177,7 +177,7 @@ describe('bodyParser.urlencoded()', function () {
       it('should parse deep object', function (done) {
         var str = 'foo'
 
-        for (var i = 0; i < 500; i++) {
+        for (var i = 0; i < 250; i++) {
```

Comment:
Nevermind... I didn't think the error would be `SyntaxError: JSON.parse Error: Invalid character at position:1`. I wonder what is going on here...